fourmolu.yaml

```diff
@@ -0,0 +1,6 @@
+indentation: 2
+column-limit: 100
```

Comment:
Enforcing a column limit has various idempotency issues. Usually minor, so no need to do sth immediately, but something to keep in mind if it gets too annoying.

Reply:
Would you rather suggest to reformat the code base and to not enforce the limit? I do not see how we would be able to prevent addition of longer lines in this case.

Reply:
No, what you do in this PR looks good to me, just pointing out this potential downside of `column-limit`, but no immediate action required.